api/internal/pkg/response.go

```diff
@@ -16,7 +16,7 @@ func ResponseOk(w http.ResponseWriter) {
 	}
 	respData, err := json.Marshal(&successResp)
 	if err != nil {
-		log.Fatal(err)
+		fmt.Println(err)
```

Comment:
please return here and process the response writer to write an error message for user to understand